### PR TITLE
[Play-311] - Add Sidebar For Visual Guidelines React Page

### PR DESCRIPTION
**Pull request by nidaqg**
#### Screens
Screen Shot 2022-09-07 at 11 30 28 AM

Screen Shot 2022-09-07 at 11 30 48 AM

#### Breaking Changes

No breaking changes, adding to playbook website only

#### Runway Ticket URL

[Runway Story](https://nitro.powerhrg.com/runway/backlog_items/PLAY-311)

#### How to test this

Test in milano env

#### Checklist:

- [ ] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [ ] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [ ] **SCREENSHOT** Please add a screen shot or two.
- [ ] **SPECS** Please cover your changes with specs.
- [ ] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

### CODE REVIEW

File: playbook-website/app/javascript/components/ReactSidebar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "Spacing",
+  "Border Radius",
+  "Typography",
+  "Shadows",
+  "Display",
+  "Cursor",
+  "Flex Box",
+];
+
+const ReactSidebar = () => {
```
Comment (thestephenmarshall) [inline playbook-website/app/javascript/components/ReactSidebar/index.tsx:19]: Could we just called this `Sidebar`? I don't know why we would need to add `React` to the naming unless I'm missing something.

File: playbook-website/app/javascript/components/VisualGuidelines/index.tsx [hunk 2]
```diff
@@ -1,50 +1,67 @@
...
       <Cursor example={examples.cursor_jsx} />
       <FlexBox example={examples.justify_self_jsx} />
     </React.Fragment>
-  )
-}
+  );
+};
 
-export default VisualGuidelines
+export default VisualGuidelines;
```
Comment (thestephenmarshall) [inline playbook-website/app/javascript/components/VisualGuidelines/index.tsx:67]: Thanks for converting the singles to doubles and adding those anchor links! 🚀

File: playbook-website/app/views/layouts/_sidebar.html.erb [hunk 3]
```diff
@@ -76,4 +76,8 @@
       </div>
     <% end %>
   <% end %>
+<% end %>
+
+<% if current_page?('/visual_guidelines_react') %>
+  <%= react_component("ReactSidebar") %>
 <% end %>
```
Comment (thestephenmarshall) [inline playbook-website/app/views/layouts/_sidebar.html.erb:83]:
```suggestion
<% end %>

```

**Review by thestephenmarshall — APPROVED** [review]
Just one comment. It is not really critical, but just wanted to better understand.

File: playbook-website/app/javascript/components/ReactSidebar/index.tsx (hunk 1 above)
Comment (nidaqg) [reply playbook-website/app/javascript/components/ReactSidebar/index.tsx:19]: Will make that change 👍🏽

**Review by augustomallmann — COMMENTED** [review]
Great job Nida!!

**Review by jasperfurniss — APPROVED** [review]

**Pull request merged**